<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ThermoHygroHistory</class>
    <widget class="QDialog" name="ThermoHygroHistory">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>温湿度历史记录</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QLabel" name="label">
                            <property name="text">
                                <string>开始时间:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QDateTimeEdit" name="startDateTime">
                            <property name="calendarPopup">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_2">
                            <property name="text">
                                <string>结束时间:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QDateTimeEdit" name="endDateTime">
                            <property name="calendarPopup">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="queryButton">
                            <property name="text">
                                <string>查询</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QCustomPlot" name="customPlot" native="true"/>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>QCustomPlot</class>
            <extends>QWidget</extends>
            <header>qcustomplot.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>